### PR TITLE
FEATURE: Show information about roles and privileges

### PR DESCRIPTION
**Pull request by daniellienert**
This feature improves the policy selection in the account editing by making use of the labels and descriptions added to roles and privileges with Neos 7.0

**Before:**
image

**After:** 
image

**Todo:**

- [x] Add some css / javascript to start with the privileges closed and make the toggleable when clicking on th (i) icon.

### CODE REVIEW

File: Neos.Neos/Resources/Private/Partials/Module/Shared/EditAccount.html [hunk 1]
```diff
@@ -1,41 +1,111 @@
...
 			<label class="neos-control-label" for="password[0]">{neos:backend.translate(id: 'password')}</label>
 			<div class="neos-controls">
-				<f:form.password name="password[0]" id="password[0]" class="neos-span12" />
+				<f:form.password name="password[0]" id="password[0]" class="neos-span12"/>
 			</div>
 		</div>
 
 		<div class="neos-control-group{f:validation.ifHasErrors(for: 'password', then: ' neos-error')}">
-			<label class="neos-control-label" for="password[1]">{neos:backend.translate(source: 'Modules', id: 'users.repeatPassword')}</label>
+			<label class="neos-control-label" for="password[1]">{neos:backend.translate(source: 'Modules', id:
```
Comment (Sebobo) [inline Neos.Neos/Resources/Private/Partials/Module/Shared/EditAccount.html:22]: Formatting went here a bit crazy.

File: Neos.Media.Browser/Configuration/Policy.yaml [hunk 2]
```diff
@@ -16,6 +16,7 @@ privilegeTargets:
       matcher: 'method(Neos\Media\Browser\Controller\AssetProxyController->(index|import)Action())'
 
     'Neos.Media.Browser:ManageAssetCollections':
+      label: Manage Asset Collections
```
Comment (Sebobo) [inline Neos.Media.Browser/Configuration/Policy.yaml:19]: Is it possible to use `i18n` labels here?

Comment (daniellienert) [reply Neos.Media.Browser/Configuration/Policy.yaml]: Not yet

File: Neos.Neos/Resources/Private/Partials/Module/Shared/EditAccount.html (hunk 1 above)
Comment (daniellienert) [reply Neos.Neos/Resources/Private/Partials/Module/Shared/EditAccount.html]: Format uncrazyfied

**Comment by daniellienert** [comment]
Reached out to @markusguenther, who likes to help with the missing toggle javascript.

**Comment by markusguenther** [comment]
I added two little components `DropDown` and `DropDownGroup`. So we can now have an effect like in the video with just some little markup changes. I did not adjust styles yet but as I add some attributes and classes we can easily transform icons of the trigger element and so on :)

Looking for feedback. When the EmberJS PR will be merged before this one, we will migrate that to TS as well.
We could also add some Sass to have a ease in and ease out effect to make it look smoother.

https://user-images.githubusercontent.com/1014126/112375266-5ff09800-8ce3-11eb-880e-dc84154fd161.mp4

**Comment by markusguenther** [comment]
The Webpack Upgrade has been merged... so we can now use the new build chain, but if we wait until the emberjs removal. We can also migrate the new component to TypeScript.

**Comment by daniellienert** [comment]
@markusguenther could you give me a hint on how to rebuild the JS to solve that conflict?

**Review by markusguenther — APPROVED** [review]
The PR works fine. Only downside are the hard coded labels.
Screenshot 2021-04-06 at 20 52 42

File: Neos.Neos/Resources/Private/Translations/en/Modules.xlf [hunk 3]
```diff
@@ -416,8 +416,14 @@
 			<trans-unit id="users.roles" xml:space="preserve">
 				<source>Roles</source>
 			</trans-unit>
+      <trans-unit id="users.roles.directlyAssignedPrivileges" xml:space="preserve">
+        <source>Directly assigned privilege</source>
+      </trans-unit>
```
Comment (albe) [inline Neos.Neos/Resources/Private/Translations/en/Modules.xlf:421]:
```suggestion
			<trans-unit id="users.roles.directlyAssignedPrivileges" xml:space="preserve">
				<source>Directly assigned privilege</source>
			</trans-unit>
```

File: Neos.Neos/Resources/Private/Translations/en/Modules.xlf [hunk 4]
```diff
@@ -416,8 +416,14 @@
...
 			<trans-unit id="users.name" xml:space="preserve">
 				<source>Name</source>
+				</trans-unit>
+      <trans-unit id="users.roles.fromParentRole" xml:space="preserve">
+        <source>From Parent Role "{role}"</source>
```
Comment (albe) [inline Neos.Neos/Resources/Private/Translations/en/Modules.xlf:426]:
```suggestion
			</trans-unit>
			<trans-unit id="users.roles.fromParentRole" xml:space="preserve">
				<source>From Parent Role "{role}"</source>
```

**Review by albe — COMMENTED** [review]
Looking good from a rough glance over everything (didn't strictly check the markup and implementation of the DropDown component). Hope tests pass now again with the fix in Flow

**Review by albe — APPROVED** [review]

**Comment by markusguenther** [comment]
Resolved the conflicts and migrated the new components to typescript as we now have the support in the master :)

**Comment by albe** [comment]
Great job everyone :) Another nice feature for Seven-One

**Pull request merged**